<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1944</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QTextEdit" name="code_editor">
      <property name="minimumSize">
       <size>
        <width>960</width>
        <height>800</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>960</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="0" column="3">
       <widget class="QLineEdit" name="breakpoint_line">
        <property name="minimumSize">
         <size>
          <width>390</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>390</width>
          <height>25</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="placeholderText">
         <string>breakpoint</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="get_desc_button">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Get docs</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+D</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="run_button">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="number_of_cores_line">
        <property name="minimumSize">
         <size>
          <width>390</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>390</width>
          <height>25</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>number of cores(def=4)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1" rowspan="2" colspan="2">
     <widget class="QTextEdit" name="text_to_show">
      <property name="minimumSize">
       <size>
        <width>960</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>960</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionasdf">
   <property name="text">
    <string>asdf</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
